src/view/shell/Drawer.tsx

```diff
@@ -262,6 +264,17 @@ let DrawerContent = ({}: {}): React.ReactNode => {
               href="https://bsky.social/about/support/privacy-policy"
               text={_(msg`Privacy Policy`)}
             />
+            {kawaii && (
+              <Text type="md" style={pal.textLight}>
+                Logo by{' '}
```

Comment:
Sorry to butt in, what about wrapping this section in `<Trans>` so that it can be translated into Japanese?